tests/framework/pvc_utils.go

```diff
@@ -0,0 +1,192 @@
+package framework
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	defaultPvcMountPath = "/pvc"
+	poll                = 2 * time.Second
+)
+
+// Creates a PVC in the passed in namespace from the passed in PersistentVolumeClaim definition.
+// An example of creating a PVC without annotations looks like this:
+// CreatePVCFromDefinition(client, namespace, NewPVCDefinition(name, size, nil))
+func (f *Framework) CreatePVCFromDefinition(namespace string, def *k8sv1.PersistentVolumeClaim) (*k8sv1.PersistentVolumeClaim, error) {
```

Comment:
Yes or if I need to create a PVC in an existing NS for whatever reason. Because a test can potentially work in multiple namespaces, basically all our utility functions will need a namespace parameter.